src/Core/src/Handlers/ScrollView/ScrollViewHandler.iOS.cs

```diff
@@ -138,9 +138,17 @@ static void UpdateContentView(IScrollView scrollView, IScrollViewHandler handler
 
 			if (scrollView.PresentedContent is { } content)
 			{
+				var contentContainer = new ContentView()
```

Comment:
How do we know if want a `ContentView` or a `GeneralWrapperView` ? Woudln't the second one handle the layout better?